lib/utils.js

```diff
@@ -13,9 +13,13 @@ exports.emitErr = function (stream, err) {
 // and all matched files. Switching options does not break Sass cache so we do
 // it ourselves. Possibly a bug: https://github.com/sass/sass/issues/1830
 exports.createIntermediatePath = function (sources, matches, options) {
+	defaults.tempSubDir = 'gulp-ruby-sass';
```

Comment:
I believe `defaults` is not available here. How about making the default `tempDir` in index.js contain the uid? Then we won't have to change this file or the clear function at all.